SwaMe.Console/Program.cs

```diff
@@ -44,7 +44,9 @@ static void Main(string[] args)
                     MaxThreads = options.MaxThreads
                 };
 
-                CheckFileIsReadableOrComplain(options.InputFile);
+                // stdin is always considered readable; anything else needs a check (#99)
+                if (!"-".Equals(options.InputFile))
```

Comment:
Magic string shared across 3 projects - very small number of locations and unlikely to change, but would prefer it if we made this evaluation in the calling assembly and passed a StdIn / StdOut bool on the read/write method call

Reply:
Agree about parsing "-" in a console app.  An alternative would be to pass null to indicate "no file, use input/output"; thoughts?

Reply:
Moved to using nulls.